tests/integration/test_register_persister.py

```diff
@@ -10,10 +10,25 @@
 from vcr.persisters.filesystem import FilesystemPersister
 
 
+class CustomFilesystemPersister(object):
```

Comment:
I'm going to merge this as is, but I think it would be much better to write a test that just asserts that the function is called with the right stuff. Doing it this way is just so much more work.